src/NuGet.Clients/NuGet.PackageManagement.UI/PackageItemLoader.cs

```diff
@@ -231,11 +231,15 @@ public async Task UpdateStateAndReportAsync(SearchResultContextInfo searchResult
             // but for project view, get the allowed version range and pass it to package item view model to choose the latest version based on that
             if (_packageReferences == null && !_context.IsSolution)
             {
-                IEnumerable<Task<IReadOnlyCollection<IPackageReferenceContextInfo>>> tasks = _context.Projects
-                    .Select(project => project.GetInstalledPackagesAsync(
-                        _context.ServiceBroker,
-                        cancellationToken).AsTask());
-                _packageReferences = (await Task.WhenAll(tasks)).SelectMany(p => p).Where(p => p != null);
+                IProjectContextInfo project = _context.Projects.SingleOrDefault();
```

Comment:
Theoretically, I don't think it's possible to have anything except 1 project when `IsSolution` is false.

Reply:
No it means whether you opened Solution PMUI or Project PMUI. VS seems to always create a solution when opening a csproj directly.